modules/src/ics02_client/client_def.rs

```diff
@@ -281,7 +284,39 @@ pub enum AnyConsensusState {
     Mock(MockConsensusState),
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Serialize)]
+// #[serde(try_from = "Timestamp", into = "Timestamp")]
+pub enum AnyTime {
```

Comment:
I don't understand why we need a need type here. This would be necessary if there was a raw (proto) type that required self-standing de/serialization and can be instantianted into different domain types (like a consensus state).